### PR TITLE
feat: added interpreter rule in static rules

### PR DESCRIPTION
**Pull request by antoinerubeiz**
## Description

Added a new rule in the staticRules.json file that raises an exception if the Interpreter Required field does not contain a 0 or a 1. 

## Context

It adds a necessary rule to the static validation function.

## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [x] I have followed the code style of the project
- [x] I have added tests to cover my changes
- [x] I have updated the documentation accordingly
- [x] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [x] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: application/CohortManager/src/Functions/ScreeningValidationService/StaticValidation/staticRules.json [hunk 1]
```diff
@@ -1,79 +1,83 @@
...
-      {
+        },
+        {
         "RuleName": "47.NewParticipantWithRemovalOrDeath",
         "Expression": "participant.RecordType != Actions.New OR (string.IsNullOrEmpty(participant.ReasonForRemoval) AND string.IsNullOrEmpty(participant.ReasonForRemovalEffectiveFromDate) AND string.IsNullOrEmpty(participant.DateOfDeath))"
-      },
-      {
+        },
+        {
+        "RuleName": "49.InterpreterCheck",
```
Comment (Warren-Pitterson) [inline application/CohortManager/src/Functions/ScreeningValidationService/StaticValidation/staticRules.json:74]: Is there a standard for RuleName? 

All the others seem to be number.PropertyName, 

I'm thinking this should follow the same and be "49.IsInterpreterRequired"

**Review by stephhou — CHANGES_REQUESTED** [review]
Just putting reminder to add unit tests, per Michael's comments.

File: application/CohortManager/src/Functions/ScreeningValidationService/StaticValidation/staticRules.json (hunk 1 above)
Comment (antoinerubeiz) [reply application/CohortManager/src/Functions/ScreeningValidationService/StaticValidation/staticRules.json]: I wasn't sure, but changed it to what you suggested!

**Review by andrewmorris43 — DISMISSED** [review]
Looks good to me. No errors when running a smoke test on my machine too.

**Review by stephhou — DISMISSED** [review]

**Review by Warren-Pitterson — APPROVED** [review]

**Pull request merged**